backend/src/schema/resolvers/passwordReset.spec.js

```diff
@@ -16,158 +21,202 @@ const getAllPasswordResets = async () => {
   return resets
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-10T08:09:50Z" title="Tuesday, September 10th 2019, 10:09:50 am +02:00">Sep 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -16,158 +21,203 @@ const getAllPasswordResets = async () => {
   return resets
```


where is this coming from?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-10T14:16:00Z" title="Tuesday, September 10th 2019, 4:16:00 pm +02:00">Sep 10, 2019</time>_

---


two lines above?